### DISCUSSION TITLE
Implement sqlite_orm for other DBMS

### DISCUSSION THREAD

**Discussion by ivanKushnarenko** [fnc12/sqlite_orm#1221] [Ideas]
It would be really cool to see sqlite_orm versions for e.g. PostgreSQL, MySQL etc.
I believe it is not hard to implement, considering that all SQL queries should work the same for different DBs.
The only part that should be changed - creating a storage, where instead of name of sqlite file, you should pass more parameters to connect to some db.
And probably the name of the library should be changed in that case too :)

**Comment by fnc12** [comment]
what DB support you expect exactly? The idea is nice and we actually are moving towards

**Reply by ivanKushnarenko** [reply]
I do not wait for some specific DB, i just wanted to know if there are such plans and how hard it is to implement

**Reply by fnc12** [reply]
ok. Postgres is being implemented right now as a dedicated repo

**Reply by ivanKushnarenko** [reply]
Do u have public repository for it? I would like to take a look

**Reply by fnc12** [reply]
@ivanKushnarenko no the repo is private. It is closed source project with paid only license. If you are going to use it on paid term we can talk by email fnc12@me.com
